pages/getting-started/developers.md

```diff
@@ -211,7 +211,12 @@ NOTE: If you plan on upgrading to newer versions of the Standards in the future,
 
 ## Notes on accessibility
 
-We’ve designed the Standards to support older and newer browsers through progressive enhancement, and they officially support Internet Explorer 9 and up, along with the latest versions of Chrome, Firefox, and Safari. Internet Explorer 8 and below generally see very low usage, and most agency websites should be able to safely begin support at Internet Explorer 9.
+- Originally the Standards were desiged to support Internet Explorer 9 and up
```

Comment:
Typo, `desiged` -> `designed`

Reply:
Er also, there should be a period at the end of this sentence I think.

Reply:
I'd add a comma after Originally